ftw/upgrade/command/__init__.py

```diff
@@ -40,17 +41,44 @@
 {t.bold}ZOPE INSTANCE DISCOVERY:{t.normal}
     The Zope instance is discovered automatically by searching for all \
 "zope.conf" files in "parts/instance*" of the buildout directory, looking up \
-the instance port and testing whether the port is bound on localhost. If multiple \
-instances are running, the first one running is used.
+the instance port and testing whether the port is bound on localhost.
+If multiple instances are running, the first one running is used.
 
 {t.bold}AUTHENTICATION:{t.normal}
-    The JSON API requires authentication as a "Manager" user by using basic \
-authentication. The authentication credentials can be passed with the \
-"--auth" argument in form "<username>:<password>" for all commands requiring \
-authentication.
-
-    Alternatively, the credentials can be set in the "UPGRADE_AUTHENTICATION" \
-environment variable which will be used as default for the "--auth" argument.
+    There are three authentication options which are tested and used in this \
+order:
+
+    - "--auth" argument for basic authentication
+    - "UPGRADE_AUTHENTICATION" environment variable for basic authentication
+    - automatic tempfile based authorization negotiation, using the \
+      automatically generated manager user "system-upgrade".
+
+    {t.bold}Tempfile negotiation authentication{t.normal}
```

Comment:
Could you list the detailed authentication option description in the same order as above in the shortlist?